tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckDates.java

```diff
@@ -806,6 +802,8 @@ public static String checkIso8601(String path, String value) {
             default:
                 return null;
         }
+
+        String intervalPosition = "1st";
```

Comment:
`RuleBasedNumberFormat(Locale.ENGLISH, RuleBasedNumberFormat.SPELLOUT).format(1, "%spellout-ordinal")`

Reply:
Not necessary for this test; we always want English.